pkgs/development/compilers/fstar/default.nix

```diff
@@ -61,6 +61,15 @@ stdenv.mkDerivation rec {
     installShellCompletion --zsh --name _fstar.exe .completion/zsh/__fstar.exe
   '';
 
+  passthru.updateScript = writeScript "update-fstar" ''
+      #!/usr/bin/env nix-shell
+      #!nix-shell -i bash -p git gnugrep common-updater-scripts
+      set -eu -o pipefail
+
+      version="$(git ls-remote --tags git@github.com:FStarLang/FStar.git | grep -Po 'v\K\d{4}\.\d{2}\.\d{2}' | sort | tail -n1)"
```

Comment:
Is this needed? It looks like the normal update scripts could handle this?

Reply:
Well, I'm not sure it's needed, the thing is, for now, `nixpkgs-update` (and hence the `r-ryantm` bot) is not able to update F* correctly (for instance: https://r.ryantm.com/log/fstar/2022-12-14.log).
It looks up directly Repology instead of trying to update via tags and fails.
(currently, the most up-to-date F* on Repology is pinned to a version that seems not to exist)

That's why I'm adding this script. But if the problem comes from somewhere else that's indeed useless.
I have no clue how to debug what `r-ryantm` does and why it looks up Repology directly instead of just taking the last github version tag.